### PR TITLE
build and deploy rust docs

### PR DESCRIPTION
**Pull request by reiner-dolp**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -93,3 +91,23 @@ script:
         return 1 ;
         ;;
     esac
+
+
+after_success: |
+    case "$TEST_KIND" in
+      comprakt-release)
+        cargo doc \
```
Comment (flip1995) [inline .travis.yml]: `--document-private-items`?

Otherwise r=me

Comment (reiner-dolp) [reply .travis.yml]: Yeah, I don't know, do we want that?

Comment (flip1995) [reply .travis.yml]: otherwise documentation of every bin crate will be empty

**Comment by reiner-dolp** [comment]
bors r=flip1995

File: .travis.yml (hunk 1 above)
Comment (flip1995) [reply .travis.yml]: image

vs only documentation for `LoweringOptions`

**Pull request merged**